hardware_frame_context.go

```diff
@@ -0,0 +1,48 @@
+package astiav
+
+//#include <libavcodec/avcodec.h>
+import "C"
+import (
+	"fmt"
+	"unsafe"
+)
+
+type HardwareFrameContext struct {
```

Comment:
Can you add this comment above `// https://github.com/FFmpeg/FFmpeg/blob/n7.0/libavutil/hwcontext.h#L115`?